added Regex util for password pattern requirements and input validati…

Kdp2016 opened the pull request:
…on messages for register/login request

Commit:
added Regex util for password pattern requirements and input validation messages for register/login request

jsparks9 committed:
fixed calls

jsparks9 reviewed (dismissed):
Looks good. I tested the regex in an online Java compiler.

jsparks9 committed:
Update Regex.java

Replaced \\d with 1-9 and added comments. I tested this regex in an online Java compiler.

jsparks9 committed:
Fixed typo

jsparks9 committed:
Merge branch 'main' of http://github.com/Revature-Capstone-1350/e-commerce-backend into inputValidation1/kalon
